brainiak/fcma/voxelselector.py

```diff
@@ -103,7 +107,7 @@ def __init__(self,
                  raw_data,
                  raw_data2=None,
                  voxel_unit=64,
-                 process_num=None,
+                 process_num=4,
```

Comment:
This should be limited by the number of CPUs, i.e., max(4, cpus).

Reply:
I think it should be min(4, cpus). Fixed.